google key

```diff
@@ -0,0 +1,3 @@
+start
+626749246571-vlg7ga8tnq9abc0fddnbppepkm4l8nbp.apps.googleusercontent.com
```

Comment:
:exclamation:**Cycode**: Secret of type: 'Google Oauth Key' was found.
Severity: Medium <!-- 0937e197-c7a1-4b45-8930-f0c88a9ef9d8 -->